Align cliname name bmc (#35134)
demisto/content: Packs/BmcITSM/ReleaseNotes/1_0_23.md

#### Incident Fields

##### Bmc Display ID

> - Fixed an issue where *BmcITSM Display ID* did not match the cli machine name.

Reviewer: ```suggestion
- Renamed the field from *BmcITSM Display ID* to *Bmc Display ID* to align with its CLI name and resolve an issue where data could not be mapped into the field.
```

#### Mappers

##### BMC ITSM Incoming Mapper

> - Updated the mapping of the field *Bmc ITSM Display ID* to the field *Bmc Display ID* for alignment with the cli name.

Reviewer: ```suggestion
- Replaced the mapping for the *Bmc ITSM Display ID* incident field with the *Bmc Display ID* incident field.
```
